src/bundle/Resources/views/Security/base.html.twig

```diff
@@ -25,7 +25,13 @@
                         {%- endblock content -%}
                     </div>
                     <div class="ez-login__footer">
-                        <span>eZ Platform ©1999-{{ "now"|date("Y") }} <a href="http://ez.no">eZ Systems AS</a> and others.</span>
+                        <span>{{ ''|trans({
```

Comment:
Translation key is missing 😉

Reply:
👼 focused too much on the link, and I got sidetracked. Thank you ^^

Reply:
Should be fixed now. Not 100% sure about the key, let me know if you have a better one.